src/main/resources/db/changelog/db.changelog-diff-20181001-1303.yaml

```diff
@@ -0,0 +1,12 @@
+databaseChangeLog:
```

Comment:
You need to add this changelog to the master change log file

Reply:
Thanks, added.